src/features/layers_panel/components/MapLayersPanel/MapLayersPanel.module.css

```diff
@@ -17,7 +17,7 @@
   /* allow resize second child which is content wrap */
```

Comment:
any change of structure broke this, what part resizable and max height should be porps